Source/Scene/Model.js

```diff
@@ -537,6 +540,18 @@ function Model(options) {
    */
   this.showOutline = defaultValue(options.showOutline, true);
 
+  /**
+   * The {@link SplitDirection} to apply to this model.
+   *
+   * @type {SplitDirection}
+   * @default {@link SplitDirection.NONE}
+   */
+  this.splitDirection = defaultValue(
```

Comment:
Should this property also be added to `ModelExperimental` at this point? CC @lilleyse?

Reply:
Sure, that would be helpful. Otherwise we can add it to https://github.com/CesiumGS/cesium/issues/9520